8968-rest-bundle-creation done (#8988)
magento/devdocs: src/guides/v2.3/rest/tutorials/bundle-product/plan-product.md

---
layout: tutorial
group: rest-api
title: Step 1. Plan the product
subtitle: Create a bundle product tutorial
menu_title: Step 1. Plan the product
menu_order: 10
level3_subgroup: bundle-product-tutorial
return_to:
  title: REST tutorials
  url: rest/tutorials/index.html
functional_areas:
  - Integration
contributor_name: Ziffity
contributor_link: https://ziffity.com/
---

> we'll create a bundle product out of the following existing simple products.

Reviewer: ```suggestion
Create a bundle product out of the following existing simple products.
```

Author: @BarnyShergold as per suggestion i have updated

SKU | Product Type
--- | ---
24-WG080 | Bundle Product
24-WG081-blue | Simple Product
24-WG082-blue | Simple Product
24-WG083-blue | Simple Product
24-WG084|  Simple Product
24-WG085 | Simple Product
24-WG086 | Simple Product
24-WG087 | Simple Product
24-WG088 | Simple Product

## Define bundle product characteristics

> The following table lists the general characteristics of quest pursuit jump cardo ball kit we're creating. These items are among those listed on the New Product page in Admin when the Gear attribute set is selected.

Reviewer: ```suggestion
The following table lists the general characteristics of the Quest Pursuit Jump Cardio Ball Kit being created. These items are among those listed on the New Product page in Admin when the Gear attribute set is selected.
```

Author: @BarnyShergold as per suggestion i have updated

Characteristic | Description
--- | ---
Status | 1
Attribute Set | Gear
Product Name | Quest Pursuit Jump Cardo Ball Kit
SKU | 24-UG00
In Stock | true
Categories | Gear, Fitness Equipment
Visibility | Catalog, Search
Description | Quest Pursuit Jump Cardo Ball Kit

> Short Description | <ul><li>Quest Strength Band brings your own personal gym with you everywhere you go</li><li>Save your knees and joints while strengthening arms, legs and core with the Pursuit</li><li>The Zing Jump Rope goes anywhere and can be used any time</li><li>Make the most of your limited workout window with our Dual-Handle Cardio Ball</li></ul>

Reviewer: ```suggestion
Short Description | <ul><li>The Quest Strength Band brings your own personal gym with you everywhere you go</li><li>Save your knees and joints while strengthening arms, legs and core with the Pursuit</li><li>The Zing Jump Rope goes anywhere and can be used any time</li><li>Make the most of your limited workout window with our Dual-Handle Cardio Ball</li></ul>
```

Author: @BarnyShergold as per suggestion i have updated

> Description | <ul><li>Quest Strength Band brings your own personal gym with you everywhere you go</li><li>Save your knees and joints while strengthening arms, legs and core with the Pursuit</li><li>The Zing Jump Rope goes anywhere and can be used any time</li><li>Make the most of your limited workout window with our Dual-Handle Cardio Ball</li></ul>

Reviewer: ```suggestion
Description | <ul><li>The Quest Strength Band brings your own personal gym with you everywhere you go</li><li>Save your knees and joints while strengthening arms, legs and core with the Pursuit</li><li>The Zing Jump Rope goes anywhere and can be used any time</li><li>Make the most of your limited workout window with our Dual-Handle Cardio Ball</li></ul>
```

Author: @BarnyShergold as per suggestion i have updated

Meta Ttile | Quest Pursuit Jump Cardo Ball Kit

> Meta Description | <p>Set of three products like  Quest Lumaflex™ Band,Pursuit Lumaflex™ Tone Band,Zing Jump Rope,Dual Handle Cardio Ball</p>

Reviewer: ```suggestion
Meta Description | <p>Set of three products like Quest Lumaflex™ Band, Pursuit Lumaflex™ Tone Band, Zing Jump Rope, Dual Handle Cardio Ball</p>
```

Author: @BarnyShergold as per suggestion i have updated
